docs/source/start.rst

```diff
@@ -167,18 +167,19 @@ Connecting to the SDK
 ~~~~~~~~~~~~~~~~~~~~~
 
 The expected locations of the QVM and Compiler endpoints are configurable in pyQuil. When running on a QMI, these
-configuration values are automatically managed so as to point to the relevant Rigetti-internal endpoints. When running
+configuration values are automatically managed so as to point to the correct endpoints. When running
 locally, these default to values reasonable for a user running local instances of the Rigetti toolchain on their laptop.
-Ideally, little-to-no work will be required for setting up this configuration environment locally or remotely, or for
+Ideally, little-to-no work will be required for setting up this configuration environment locally or from your QMI, or for
 transferring functioning code from one configured environment to another.
 
 .. note::
     A config file is not necessary to run locally, though it may be useful in configuring your local set-up.
 
-In general, these values are read out of a pair of configuration files (stored at the location described by the environment
-variables ``FOREST_CONFIG`` and ``QCS_CONFIG``, or else at the respective default locations ``~/.forest_config`` and
-``~/.qcs_config``), which by default have the following respective contents:
+In general, these values are read out of a pair of configuration files which are stored at the location described by the environment
+variables ``QCS_CONFIG`` and ``FOREST_CONFIG``, or else at the respective default locations ``~/.qcs_config`` and
+``~/.forest_config``. By default, they contain the following on a local installation:
```

Comment:
Are `~/.forest_config` and `~/.qcs_config` automatically populated when you download the latest SDK? I remember having to make these files myself. Have things changed?

Reply:
The SDK does not add config files.

Reply:
Thanks, I'm going to re-write a lot of this page to be accurate (from my experiences).